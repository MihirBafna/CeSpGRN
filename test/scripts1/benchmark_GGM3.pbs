#PBS -N benchmark_GGM3
#PBS -q hive-himem
#PBS -l nodes=1:ppn=4
#PBS -l walltime=36:20:00
#PBS -j oe
#PBS -o benchmark_GGM3.out
#PBS -m abe                            
#PBS -M zzhang834@gatech.edu  

cd $PBS_O_WORKDIR
module load anaconda3
conda activate myenv
python benchmark_GGM3.py