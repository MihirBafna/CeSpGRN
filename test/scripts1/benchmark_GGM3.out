---------------------------------------
Begin PBS Prologue Tue Sep 21 21:44:02 EDT 2021
Job ID:     7016759.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   benchmark_GGM3
Queue:      hive-himem
End PBS Prologue Tue Sep 21 21:44:02 EDT 2021
---------------------------------------
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
0.01_1_0.01_1.7
0.01_1_0.1_1.7
0.01_1_1_1.7
0.1_1_0.01_1.7
0.1_1_0.1_1.7
0.1_1_1_1.7
1_1_0.01_1.7
1_1_0.1_1.7
1_1_1_1.7
10_1_0.01_1.7
10_1_0.1_1.7
10_1_1_1.7
---------------------------------------
Begin PBS Epilogue Tue Sep 21 22:25:27 EDT 2021
Job ID:     7016759.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   benchmark_GGM3
Resources:  nodes=1:ppn=4,walltime=36:20:00,neednodes=1:ppn=4
Rsrc Used:  cput=00:41:17,vmem=2048020kb,walltime=00:41:25,mem=156936kb,energy_used=0
Queue:      hive-himem
Nodes:     
atl1-1-01-018-38.pace.gatech.edu
End PBS Epilogue Tue Sep 21 22:25:28 EDT 2021
---------------------------------------
